src/parsers/manifest/smooth/create_parser.ts

```diff
@@ -630,4 +643,28 @@ function createSmoothStreamingParser(
   return parseFromDocument;
 }
 
+/**
+ * @param {string} systemId - Hex string representing the CDM, 16 bytes.
+ * @param {Uint8Array|undefined} privateData - Data associated to protection
+ * specific system.
+ * @returns {Uint8Array}
+ */
+function createPSSHBox(
+  systemId : string,
+  privateData : Uint8Array = new Uint8Array(0)
```

Comment:
Why making the privateData optionnal ?
It seems that there is always a privateData when we want to create a PSSH box in Smooth, no ?

Reply:
You're right, it was here before when it was also possible to communicate keyIds, but it makes not much sense now.

Reply:
Done.